transform/app/transform/schemas/properties/data.py

```diff
@@ -87,3 +96,19 @@
 KEYWORDS_TG = "keywords_tg"
 TAG_LIST_TG = "tag_list_tg"
 EOSC_IF_TG = "eosc_if_tg"
+
+# RELATIONS RELATED PROPERTIES
+
+# HELPER COLUMN NAMES
+DIRECTORY = "directory"
```

Comment:
Are they really useful? I mean all of the below. If not, please delete

Reply:
Moved to `app.tranform.schemas.properties.data.py`